docs/examples/use_cases/pytorch/single_stage_detector/src/coco_pipeline.py

```diff
@@ -15,110 +15,84 @@
 
 import torch
 from nvidia.dali.pipeline import Pipeline
-import nvidia.dali.ops as ops
 import nvidia.dali.types as types
+import nvidia.dali.fn as fn
+
+
+def get_coco_pipeline(default_boxes, args, seed):
```

Comment:
```suggestion
def create_coco_pipeline(default_boxes, args, seed):
```
or
```suggestion
def build_coco_pipeline(default_boxes, args, seed):
```
`get` suggests that this pipeline already exists somewhere - and we're creating new one each time this function is called.

Reply:
Done